### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by portlek**
Part of #81

### CODE REVIEW

**Comment by MrNemo64** [comment]
Is that all that's needed? What and how does this do?

**Comment by portlek** [comment]
`Part of`

**Comment by MrNemo64** [comment]
Which part is this pr addressing?

**Comment by portlek** [comment]
Updating dependencies.
image

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: maven
+    directory: "/"
+    labels:
+      - dependencies
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 99
```
Comment (MrIvanPlays) [inline .github/dependabot.yml:9]: wb 10?

Comment (portlek) [reply .github/dependabot.yml:9]: it doesn't matter but ok.

**Review by MrIvanPlays — APPROVED** [review]
I have experience using dependabot and its all good. 

@lokka30 merging if you approve.

File: .github/dependabot.yml (hunk 1 above)
Comment (MrNemo64) [reply .github/dependabot.yml]: > wb 10?

What's the point of limiting it?

Comment (MrIvanPlays) [reply .github/dependabot.yml]: > > wb 10?
> 
> What's the point of limiting it?

treasury won't have _that_ much dependencies, and I personally don't want the PR tab fulfilled top to bottom with PRs from dependabot.

Comment (MrNemo64) [reply .github/dependabot.yml]: Oh, I see

Comment (MrNemo64) [reply .github/dependabot.yml]: Now I understand what you ment

**Pull request merged**